CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ Remember to bring your dependencies up to date with `./scripts/venvinstall.sh` w
   This makes all `or_...` filters a lot more useful, but it may mean that some other filters will need some additional error handling.  
   This will require some experimentation, reporting errors in our GitHub issues for this is greatly appreciated.
 - Minor: Add "Wide Emote Limit" module (#2064)
+- Minor: Separate the timeout in the ASCII module into online & offline timeouts (#2072)
```

Comment:
Make a note in the changelog entry that previously configured timeout durations for the ASCII module will have been reset